maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/InputTextRouteHandler.swift

```diff
@@ -62,12 +62,6 @@ class InputTextRouteHandler : RouteHandler {
         }
     }
 
-    private func typingFrequencyFor(text: String) -> Int {
-        let count = max(text.count, 5)
-        return count * 2
-    }
-
-
```

Comment:
We decided to make typing frequency dependent on the text length because some users enter long strings of text.

Reply:
I think we should always type with maximum reliable frequency. If typing faster causes characters to be skipped - that would be a big problem.

Reply:
How certain are we that the typing frequency causes the skipping?

Reply:
I have high confidence that it does have an impact, for two reasons:

- We had a similar problem before with idb and adjusting the frequency helped to solve it
- This problem resurfaced after moving to the new driver, meaning it is highly likely related